docs/learning/backend/CodeService.md

```diff
@@ -125,12 +124,76 @@ Generally it is a good idea to reserve a group of codes together rather than one
 - `reserveCode` to reserve a single code
 - `reserveCodes` to reserve a set of codes
 
+Here are the steps to follow.
+
+First, you must choose the code service that should be used at the time that you create your iModel. The identity of the code service and information about how to connect  are stored in the iModel persistently.
+
+When you open a BriefcaseDb for that iModel, it will read the configuration data from the iModel and automatically try to connect to the code service. To support that, apps that access the iModel must provide the client for the code service and set `CodeService.createForIModel` to point to it.
+
+For example, if you configure your new iModel to use Bentley's cloud-based code service, apps must provide the `ITwinCodeService` client. To do that, an app simply depends on the `@bentley/code-service` package and then imports it.
+
+```ts
+import { ITwinCodeService } from "@bentley/code-service";
```

Comment:
This is currently an internal package so can't be in our public facing docs. Can you take out the section talking about this as an example?

Reply:
I made the requested changes

Reply:
> How will the apps that our customers write deal with internal codes?

They will use iTS.